components/avPublisher/avPublisher.c

```diff
@@ -207,7 +201,6 @@ static void HandleAvPushComplete
     switch (status)
     {
         case LE_AVDATA_PUSH_SUCCESS:
-
```

Comment:
This is debatable.

Reply:
What value is added by a blank line after a case statement?  Blank lines reduce the amount of code that can fit on a screen.  I agree they make sense as hints to the reader when they break up logical blocks, but I don't think that's the case with a blank after a case statement.

Reply:
The value of blank lines is that they visually break up the code into more easily human-parseable chunks.

Regardless, this is a style preference, not a defect.

Reply:
The case statement is logically coupled to the code that follows.  I agree with leaving space between the end of one case statement and the beginning of another because those are logically two different blocks.  To some extent it is a preference, but it's a preference that I justify by trying to make the code easier to understand by using blank lines only when necessary to break apart unrelated chunks.  Every blank line is one less line of code that can fit on your screen.